src/Angor/Client/Pages/Penalties.razor

```diff
@@ -132,8 +132,10 @@ else
         public string RecoveryTransactionId;
         public bool IsReleased;
         public bool IsExpired;
-        public decimal Amount;
+        public long AmountInSatoshis;
         public int DaysLeftForPenalty;
+
+        public decimal AmountInBTC => Money.Satoshis(AmountInSatoshis).ToUnit(MoneyUnit.BTC);
```

Comment:
ah I see you make the conversion here ok, even though I would prefer to convert on the page it is then more clear